doctrine.rst

```diff
@@ -53,7 +53,7 @@ The database connection information is stored as an environment variable called
     # DATABASE_URL="sqlite:///%kernel.project_dir%/var/app.db"
 
     # to use postgresql:
-    # DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8"
+    # DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=12.19 (Debian 12.19-1.pgdg120+1)&charset=utf8"
```

Comment:
This is the correct format according to [Doctrine DBAL](https://www.doctrine-project.org/projects/doctrine-dbal/en/4.0/reference/configuration.html#postgres), but I'm not sure if the `DATABASE_URL` can contain spaces and parentheses...

Can someone confirm that this works? Otherwise I will set up a test installation of Symfony with PostgreSQL and test it myself...

Reply:
for mariadb its `mariadb-10.11.5`
i dont know for postgresql, but perhaps there is sort of same convention naming

Reply:
That actually changed as well with DBAL 4.0, see [UPGRADE.md](https://github.com/doctrine/dbal/blob/4.0.x/UPGRADE.md#bc-break-removed-mariadb--prefix-hack):

> ## BC BREAK: Removed mariadb- prefix hack
> 
> Previously, it was necessary to prefix the serverVersion parameter with mariadb- when using MariaDB. Doing so is now considered invalid, and you should prefer using the version as returned by SELECT VERSION();
>
> ```diff
> -'serverVersion' => 'mariadb-10.9.3'
> +'serverVersion' => '10.9.3-MariaDB-1'
> ```

Reply:
I wasnt aware of this, still using v3
but thx for sharing :)

Reply:
With
```
DATABASE_URL="postgresql://app:!ChangeMe!@127.0.0.1:5432/app?serverVersion=12.19 (Debian 12.19-1.pgdg120+1)&charset=utf8"
```

Parsed DNS looks good in `vendor/doctrine/doctrine-bundle/src/ConnectionFactory.php`

![image](https://github.com/symfony/symfony-docs/assets/9253091/bcda2b3e-26f7-4d4e-accb-07875424469b)

Reply:
Antoine, thanks for double checking this!